### PR TITLE
Translation fixes

### PR DESCRIPTION
**Pull request by stephenwf**
* Adds various missing translations
* Fixes navigation translations

### CODE REVIEW

File: repos/elucidate/view/elucidate/activity/user-activity.twig [hunk 1]
```diff
@@ -17,17 +19,17 @@
...
                     <div class="c-dig-grid">
                         <div class="c-dig-grid__col--2">
-                            {# @todo timeAgo to be added #}
                             {{ annotation.timeAgo }}
                         </div>
                         <div class="c-dig-grid__col--10">
                             {# @todo motiviation to be added #}
                             {% if(annotation.label or canvas.label) %}
-                                Added a <strong>{{ annotation.motivation }}</strong> annotation to
+                                {{ translate('Added a %s annotation to') | format(annotation.motivation) }}
```
Comment (garyttierney) [inline repos/elucidate/view/elucidate/activity/user-activity.twig:27]: Hm, just noticed we give the wrong inflection here for motivations starting with a vowel. Not sure how you could translate that, though. LGTM

**Review by garyttierney — APPROVED** [review]

File: repos/elucidate/view/elucidate/activity/user-activity.twig (hunk 1 above)
Comment (stephenwf) [reply repos/elucidate/view/elucidate/activity/user-activity.twig:27]: Yeah Annotation motivations could be any, Ill maybe switch it over to `added an annotation (%s) to`

File: repos/annotation-studio/src/Controller/CaptureModelController.php [hunk 2]
```diff
@@ -112,6 +112,45 @@ public function captureModelAction()
         );
     }
 
+    public function translationAction() {
+        $site = $this->currentSite();
+        if ($site) {
+            $file = __DIR__ . '/../../../../translations/s/' . $site->slug() . '/annotation-studio.json';
+            if (file_exists($file)) {
```
Comment (garyttierney) [inline repos/annotation-studio/src/Controller/CaptureModelController.php:119]: Should be `is_file($file)`. `file_exists(...)` will return `true` for directories too.

**Review by garyttierney — CHANGES_REQUESTED** [review]

File: repos/annotation-studio/src/Controller/CaptureModelController.php (hunk 2 above)
Comment (stephenwf) [reply repos/annotation-studio/src/Controller/CaptureModelController.php:119]: Ah good catch!

**Review by garyttierney — APPROVED** [review]

**Pull request merged**